pkg/v1/providers/bootstrap-kubeadm/v1.1.3/bootstrap-components.yaml

```diff
@@ -47,58 +47,42 @@ spec:
         description: KubeadmConfig is the Schema for the kubeadmconfigs API.
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation
-              of an object. Servers should convert recognized schemas to the latest
-              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Comment:
General questions about all the modified files in pkg/v1/providers (and I understand this is a one time thing to sync the files to what is generated). Have we verified that there are no material differences introduced? If not, probably something like a 2 file diff with kapp cli can help here.

Reply:
We did do extensive diffing to ensure all of these. All the noise is super unfortunate. 

We diff'd the all the contents of the `hack/provider-sync-tools/<capi-package>/render directory against the content of the `pkg/v1/provider/<capi-folder>` by running them both through ytt first. This eliminated the majority of the diff, and what remained seems legit.